src/routes/phenomenon.route.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const app = express();
+
+router.get("/", (res, req) => {
+    res.status(200).send("Phenomenon page");
+        });
```

Comment:
Mind your indentation :P

Reply:
Yes, ewww. I need to fix that. Thank you!